server/cmd/mmctl/commands/post.go

```diff
@@ -85,8 +86,13 @@ func init() {
 
 func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	message, _ := cmd.Flags().GetString("message")
+	// len(args)>1 makes sure that args[1] exists.
+	if len(args) > 1 && args[1] != "" {
+		message = args[1]
+	}
+
 	if message == "" {
-		return errors.New("message cannot be empty")
+		return fmt.Errorf("message cannot be empty")
```

Comment:
Why that change?

Reply:
No, just that we agreed somewhere that we should format error strings with fmt.Errorf

Reply:
I don't think there is such an agreement. Let's just keep the diff simple and move that discussion of of the PR.

Reply:
Okay